src/app/sessions/containers/sessions-listings-search/sessions-listings-search.component.ts

```diff
@@ -179,7 +181,7 @@ export class SessionsListingsSearchComponent implements OnInit {
     }
 
     private checkIfOnlyOneJudgeSelected() {
-        if (this.selectedHearing.numberOfSessionsNeeded === 1) {
+        if (this.selectedHearing.numberOfSessions === 1) {
```

Comment:
This check could be now changed to isMultiSession.

Reply:
Done